templates/etc/check_mk/multisite.d/wato/users.mk.j2

```diff
@@ -0,0 +1,30 @@
+{% for user in checkmk_server__multisite_users|list|d([]) %}
+{#
+ # Strip Ansible-only properties from user definitions
+ #}
+{%   for property in checkmk_server__ansible_user_properties|d([]) %}
+{%     if property in checkmk_server__multisite_users[user]|list %}
+{%       set _ = checkmk_server__multisite_users[user].pop(property) %}
+{%     endif %}
+{%   endfor %}
+{#
+ # Make sure default arguments are defined
+ #}
+{%   for key, value in checkmk_server__multisite_user_defaults.iteritems() %}
+{%     if user not in checkmk_server_fact_local_users|d({}) and key not in checkmk_server__multisite_users[user]|d({}) %}
+{%       set _ = checkmk_server__multisite_users[user].update({key: value}) %}
+{%     endif %}
+{%   endfor %}
+{% endfor %}
+{#
+ # Combine fact users with multisite users
+ #}
+{% set users = checkmk_server_fact_local_users|d({}) | combine(checkmk_server__multisite_users, recursive=True) %}
+# Written by Multisite UserDB
```

Comment:
CMK ignores comments so I guess we can proudly throw a "Written by Ansible in there" but `ansible_managed` would be a bit too much I guess :wink: